src/actions/brew.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import type {Brew} from '../database/brew';
+
+// brew action
+// so yes, this is just copying what actions/coffee.js is doing...
+
+// Are there other things besides id that we need to add? 
+
+
+export const receiveBrew = (payload: Brew, id: string) => ({
+  type: 'RECEIVE_BREW',
+  payload,
+  meta: {
+    id,
```

Comment:
in the interest of disclosure, i just want to point out that not everyone does it this way, some people will extract the id from the payload directly and the action will just look like
```
{type: 'STRING', payload}